bootstrap/lumen.php

```diff
@@ -80,6 +81,11 @@
 |
 */
 
+// Configure CORS package
```

Comment:
You could probably move this to just after `app->withEloquent`. Otherwise its a bit weird that its in the 'Register Service Provider' block. I actually think semi-standard lumen setup was to have config calls at the start of the bootstrap for issues like this